docs/examples/plot_discontinuous_tracking.py

```diff
@@ -0,0 +1,91 @@
+"""
+Discontinuous Tracking
+======================
+
+Example of a custom Mount class.
+"""
+
+# %%
+# Many real-world tracking arrays adjust their position in discrete steps
+# rather than through continuous movement. This example shows how to model
+# this discontinuous tracking by implementing a custom Mount class.
+
+import pvlib
+from pvlib import tracking, pvsystem, location, modelchain, iotools
+from pvlib.temperature import TEMPERATURE_MODEL_PARAMETERS
+import matplotlib.pyplot as plt
+import pathlib
+
+
+# %%
+# We'll define our custom Mount by extending
+# :py:class:`~pvlib.pvsystem.SingleAxisTrackerMount` for convenience.
+# Another approach would be to extend ``AbstractMount`` directly; see
+# the source code of :py:class:`~pvlib.pvsystem.SingleAxisTrackerMount`
+# and :py:class:`~pvlib.pvsystem.FixedMount` for how that is done.
+
+
+class DiscontinuousTrackerMount(pvsystem.SingleAxisTrackerMount):
+    # inherit from SingleAxisTrackerMount so that we get the
+    # constructor and tracking attributes (axis_tilt etc) automatically
+
+    def get_orientation(self, solar_zenith, solar_azimuth):
+        # Different trackers update at different rates; in this example we'll
+        # assume a relatively slow update interval of 15 minutes to make the
+        # effect more visually apparent.
+        zenith_subset = solar_zenith.resample('15min').first()
+        azimuth_subset = solar_azimuth.resample('15min').first()
+
+        tracking_data_15min = tracking.singleaxis(
+            zenith_subset, azimuth_subset,
+            self.axis_tilt, self.axis_azimuth,
+            self.max_angle, self.backtrack,
+            self.gcr, self.cross_axis_tilt
+        )
+        # propagate the 15-minute positions to 1-minute stair-stepped values:
+        tracking_data_1min = tracking_data_15min.reindex(solar_zenith.index,
+                                                         method='ffill')
+        return tracking_data_1min
+
+
+# %%
+# Let's take a look at the tracker rotation curve it produces:
+
+# a simple weather dataset for illustration -- one day of 1-minute weather
+DATA_DIR = pathlib.Path(pvlib.__file__).parent / 'data'
```

Comment:
Is the downward spike due to DNI momentarily dropping to 0 in the weather data file? While it might be nice to highlight the `read_bsrn` function here, the spike is the most noticeable thing in the results. My concern is that the people that would most benefit from this example are also the people most likely to be distracted by that spike.

Reply:
I don't like that spike either, but that was the only clear-sky day in the file.  Would it be better to keep it simple and do a quick clear-sky simulation instead of using the BSRN data?

Reply:
I'd say yes but it's not a clear win and I certainly don't insist on it.